solidity/scripts/ci-provision-external-contracts.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+set -ex
+
+# Fetch addresses of contacts migrated from keep-network/keep-core project.
+# CONTRACT_DATA_BUCKET and ETH_NETWORK_ID should be passed as environment
+# variables straight from the CI context.
+
+if [[ -z $CONTRACT_DATA_BUCKET || -z $ETH_NETWORK_ID ]]; then
+  echo "one or more required variables are undefined"
+  exit 1
+fi
+
+if ! [ -x "$(command -v jq)" ]; then echo "jq is not installed"; exit 1; fi
+
+: ${CONTRACT_DATA_BUCKET_DIR:=keep-core}
```

Comment:
Does nothing beyond expanding arguments and performing redirections ([source](https://www.gnu.org/software/bash/manual/html_node/Bourne-Shell-Builtins.html#Bourne-Shell-Builtins). This is needed to prevent running the resolved value of `CONTRACT_DATA_BUCKET_DIR`.